test/org/sagebionetworks/bridge/hibernate/HibernateAccountDaoTest.java

```diff
@@ -1924,7 +1897,6 @@ public void editAccountFailsAcrossSubstudies() throws Exception {
         
         HibernateAccount hibernateAccount = makeValidHibernateAccount(false, false);
         hibernateAccount.setAccountSubstudies(ACCOUNT_SUBSTUDIES);
-        when(mockHibernateHelper.getById(any(), any())).thenReturn(hibernateAccount);
```

Comment:
Mock wasn't called because it was mocking the wrong thing. Fixing this caused test to fail because we're not filtering in editAccount (or AccountDao) anymore.

I added this filtering back in to fix the text. This seems reasonable because sometimes we want to retrieve accounts across substudies (email notifications that someone has signed up with your credentials) but we never want to write across substudies unless you're an admin. That's a normal update operation.